services/docker-compose.yml

```diff
@@ -312,6 +429,7 @@ services:
     networks:
       - default
       - interactive_services_subnet
+
   wb-db-event-listener:
```

Comment:
Before these service and the one below inherited all their configuration from the webserver.
Since this server has only a limited amount of plugins which actually need to be configured for it to work, all those env vars are not that useful.

Since you are doing this change, could we maybe just put only the required env vars for this service instead of all the ones from the webserver and drop this inheritance mechanism? I think this will make it more simple to read and figure out what should be present at runtime on the service.

@sanderegg @pcrespov any input form you here as well?

Reply:
I guess that if we remove the env_file in the webserver service it should also be the case in these 2 services for consistency.
I am not sure which solution is the most compact in terms of inheritating the values or setting them.